src/Javis.jl

```diff
@@ -144,6 +147,10 @@ function preprocess_frames!(objects::Vector{<:AbstractObject})
         append!(frames, collect(get_frames(object)))
     end
     frames = unique(frames)
+    if !(frames ⊆ CURRENT_VIDEO[1].background_frames)
+        @warn("Some of the frames don't have a background: In this case: ",
+             setdiff(frames, CURRENT_VIDEO[1].background_frames))
+    end
```

Comment:
I feel that the warning is a bit verbose:

```
┌ Warning: Some of the frames don't have a background: In this case:
│   setdiff(frames, (CURRENT_VIDEO[1]).background_frames) =
│    1-element Array{Int64,1}:
│     11
└ @ Javis /home/src/Projects/javis/src/Javis.jl:151
```

Perhaps instead we could do this: 

```julia
        @warn("Some of the frames don't have a background: In this case: $(
             setdiff(frames, CURRENT_VIDEO[1].background_frames))")
```

which I think produces the nicer warning:

```
┌ Warning: Some of the frames don't have a background: In this case: [11]
└ @ Javis /home/src/Projects/javis/src/Javis.jl:151
```